src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -270,7 +270,7 @@
 
     <xsd:complexType name="workflow">
         <xsd:sequence>
-            <xsd:element name="initial-place" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+            <xsd:element name="initial-marking" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
both element should be here to support the BC, no ?

Reply:
Yes